### PR TITLE
GET /users endpoint by organization ID

### PR DESCRIPTION
**Pull request by erozonachi**
### What this PR does?

- Implements get users by organization ID for the web app
- Exposes [GET] /api/organizations/:id/users route
- Exposes endpoint that takes an organization ID as a request parameter 
   and a JWT token in the authorization header

### Screenshots
image

### CODE REVIEW

**Comment by samarv** [comment]
@erozonachi  can you resolve these merge conflicts

File: data/dbModels/users.js [hunk 1]
```diff
@@ -8,13 +8,26 @@ async function create(user) {
   return result[0];
 }
 
-function readBySlackId(slack_mem_id) {
-  return db('users')
-    .where({ slack_mem_id })
-    .first();
+function read(slack_mem_id = null, org_id = null) {
```
Comment (ogwurujohnson) [inline data/dbModels/users.js]: Are you aware that changes like this (renaming variable names) break things? stick to a name and have it work, if there is going to be a change communicate properly, please

File: data/dbModels/users.js [hunk 2]
```diff
@@ -8,13 +8,26 @@ async function create(user) {
...
+  if (slack_mem_id) {
+    return db('users')
+      .where({ slack_mem_id })
+      .first();
+  }
+
```
Comment (ogwurujohnson) [inline data/dbModels/users.js]: I'm not comfortable having this read user also handle read organization users, please can we adhere to the single responsibility principle we adopted earlier, get back the model how it was and create another model for this read users by organization, thanks

Comment (erozonachi) [reply data/dbModels/users.js]: Someone already started work on this

**Pull request merged**